net: lib: nrf_cloud: Clean up KConfigs so they don't clutter unrelated samples (#20074)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/nrf_cloud_coap.rst

Configuration
*************

> Enable :kconfig:option:`CONFIG_NRF_CLOUD` and configure the :kconfig:option:`CONFIG_NRF_CLOUD_COAP` option to enable or disable the use of this library.

Reviewer: ```suggestion
To enable this library, set the :kconfig:option:`CONFIG_NRF_CLOUD` and :kconfig:option:`CONFIG_NRF_CLOUD_COAP` Kconfig options.
```

Reviewer: And continue the same way with all remaining ones.
